web/src/settings_preferences.ts

```diff
@@ -70,7 +87,9 @@ function org_notification_default_language_modal_post_render() {
 
             const $link = $(e.target).closest("a[data-code]");
             const setting_value = $link.attr("data-code");
+            assert(setting_value !== undefined);
             const new_language = $link.attr("data-name");
+            assert(new_language);
```

Comment:
Is this ambiguous truthiness check guarding against `undefined` or `""`? Always be specific.

Reply:
Fixed it.

Feedback : _Always be specific._

Sure!!